src/Orleans.Core.Abstractions/Placement/ActivationCountBasedPlacement.cs

```diff
@@ -6,18 +6,5 @@ namespace Orleans.Runtime
     internal class ActivationCountBasedPlacement : PlacementStrategy
     {
         internal static ActivationCountBasedPlacement Singleton { get; } = new ActivationCountBasedPlacement();
-
-        private ActivationCountBasedPlacement()
-        {}
-        
-        public override bool Equals(object obj)
-        {
-            return obj is ActivationCountBasedPlacement;
-        }
-
```

Comment:
Yeah, this was just useless noise.  Ran across it when trying to understand how strategies where used.  Killed it because it just created confusion and was pointless.